pkg/oci/client/client.go

```diff
@@ -20,33 +20,36 @@ import (
 	"crypto/x509"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"net"
 	"net/http"
 	"net/url"
 	"os"
 	"time"
 
-	"github.com/golang/glog"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+
```

Comment:
Confusingly `github.com/oracle/oci-go-sdk` is actually an 3rd partly lib rather than internal and as such should be grouped with zap and errors etc.